### PR TITLE
Module1: Mobile menu

### PR DESCRIPTION
**Pull request by Shedrack-Sunday**
## Milestone Attained 

When the user clicks (or taps) the hamburger button, the mobile menu appears.

When the user clicks (or taps) the close (X) button, the mobile menu disappears.

When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.

When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,9 +24,32 @@
...
+          </li>
+          <li>
+              <a href="#" class="link-about">
+                  About
+              </a>
+          </li>
+          <li>
+              <a href="#" class="link-contact">
+                  Contact
+              </a>
```
Comment (sja-thedude) [inline index.html]: - [x] Kindly make sure to display the correct part of the page when the menu item is clicked as per the loom video attached.

File: style.css [hunk 2]
```diff
@@ -392,6 +392,12 @@ li {
   margin: 24px 30px 24px 30px;
 }
 
+.header-nav-mobile > button {
+  margin: 0.5em 0.5em;
+  align-self: flex-end;
+  cursor: pointer;
+}
+
```
Comment (sja-thedude) [inline style.css:400]: - [x] Kindly display the hamburger as the attached loom video explains.

File: style.css [hunk 3]
```diff
@@ -484,6 +490,56 @@ li {
...
+  top: 0%;
+  right: 0%;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  background-color: white;
+  color: #3a4a42;
+  border: 1px lightgrey solid;
+  border-radius: 5px;
```
Comment (sja-thedude) [inline style.css]: - [ ] Kindly make sure to display the popup menu and the delete button as per the Figma design shown in the attached loom video.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Shedrack-Sunday, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Correct Gitflow
✔️ All linters passing

## Required 
- [ ] Kindly update your readme with the contact details of you and your partner.
- [ ] Kindly make sure to make changes as per the loom video attached [here](https://www.loom.com/share/ee7d2d4a9d66417e9f8cd624590e5187)
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @Shedrack-Sunday  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Made changes as requested by the previous code reviewer 🏄 
- Linter checks passing ✔️ 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**